### PR TITLE
docs(swagger): add docs of password reset [#184361212]

### PR DESCRIPTION
**Pull request by SergeInGithub**
# Description

This will Add swagger documentation of the password reset via email feature.

Fixes # (issue)
#184361212

# Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have created a loom video for the new feature or enhancement
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by shyakadavis — DISMISSED** [review]
LGTM 🏄🏽

**Review by emilenfc — DISMISSED** [review]
LGTM

File: src/routes/auth.routes.ts [hunk 1]
```diff
@@ -20,9 +20,9 @@ authRouter.get("/logout", [isAuth], logoutUser); //logout a user
 
 authRouter.get("/confirm/:confirmationCode", confirmUser); // Confirm the user who registered
 
-authRouter.post("/auth/requestResetPassword", resetPasswordRequestController); // Request a reset password
+authRouter.post("/requestResetPassword", resetPasswordRequestController); // Request a reset password
```
Comment (mucyoalexaxel) [inline src/routes/auth.routes.ts:23]:
```suggestion
authRouter.post("/requestResetPassword", resetPasswordRequestController);
```

**Review by mucyoalexaxel — DISMISSED** [review]

**Review by shyakadavis — DISMISSED** [review]
LGMT 🚀

**Review by emilenfc — APPROVED** [review]
LGTM

**Pull request merged**